plugins/python/redisgears_python.c

```diff
@@ -5544,8 +5546,8 @@ static int PythonConfig_GetStrVal(const char* val, const char* defaultVal, const
 #define ForeceDownloadDepsOnEnterpriseConfigName "ForeceDownloadDepsOnEnterprise"
 #define PythonAttemptTracebackConfigName "PythonAttemptTraceback"
 #define PythonInstallReqMaxIdleTimeConfigName "PythonInstallReqMaxIdleTime"
-#define DependenciesUrlConfigName "DependenciesUrl"
-#define DependenciesSha256ConfigName "DependenciesSha256"
+#define GearsPythonUrlConfigName "GearsPythonUrl"
+#define GearsPythonSha256ConfigName "GearsPythonSha256"
```

Comment:
The name itself can not be changed cause it breaks backword compatability.

Reply:
I'll revert that.